typescript/index.d.ts

```diff
@@ -250,6 +255,26 @@ interface RaygunStatic {
      * Track Single Page Application route events.
      */
     trackEvent(type: "pageView", options: { path: string }): void;
+
+    /**
+     * Records a manual breadcrumb with the given message and metadata passed.
+     */
+    recordBreadcrumb(message:string,metadata?:any): void;
```

Comment:
It is. I decided to keep it more simple in the interest of time but that can/should be added in the future.